gpflow/kernels/mo_kernels.py

```diff
@@ -130,7 +130,18 @@ def K_diag(self, X, full_output_cov=False, presliced=False):
             stacked) if full_output_cov else stacked  # [N, P, P]  or  [N, P]
 
 
-class SeparateMixedMok(Mok, Combination):
+class IndependentLatentBase(Mok, metaclass=abc.ABCMeta):
```

Comment:
If we have `Mok` in the class name everywhere else would do that here for consistency, too

Reply:
I'm sorting out naming in the next PR, on Artem's request.